hack/verify-golint.sh

```diff
@@ -46,8 +46,10 @@ export IFS=$'\n'
 # with a leading underscore. We'll need to support both scenarios for all_packages.
 all_packages=()
 while IFS='' read -r line; do all_packages+=("$line"); done < <(go list -e ./... | grep -vE "/(third_party|vendor|staging/src/k8s.io/client-go/pkg|generated|clientset_generated)" | sed -e 's|^k8s.io/kubernetes/||' -e "s|^_\(${KUBE_ROOT}/\)\{0,1\}||")
+# The regex below removes any "#" character and anything behind it and including any
+# whitespace before it. Then it removes empty lines.
```

Comment:
imo we should require no whitespace before it given how simple these files are

Reply:
oh I see, the intention is to put it on the same line as the package.
I guess this is fine since go packages likely cannot contain `#` in their path, otherwise I'd suggest having comment _lines_ that start with `#` exactly.

Reply:
Thanks for reviewing, @BenTheElder!

Yes, the idea is that the comments are inline with the package paths. The file becomes more compact, so it's easier to see what paths are related. If the file is ever automatically sorted, which [kube::util::check-file-in-alphabetical-order](https://github.com/kubernetes/kubernetes/blob/master/hack/lib/util.sh#L712-L727) will instruct you to do if it's not, the comment will stay with the package name.

What do you think is the best action to take here? I'm happy to make any changes :)

Reply:
IMO this is the best option, I would have marked these comments resolved, but I can't :(

Reply:
Goodie. Thanks 🙌